quilt/test/test_command.py

```diff
@@ -306,3 +306,34 @@ def callback(request):
             command.build('foo/bar', build_path)
 
             self.requests_mock.reset()  # Prevent the "not all requests ..." assert.
+
+    def test_rm(self):
+        """
+        Test removing a package.
+        """
+        mydir = os.path.dirname(__file__)
+        build_path = os.path.join(mydir, './build_simple.yml')
+        command.build('foo/bar', build_path)
+
+        command.rm('foo/bar', force=True)
+        teststore = store.PackageStore(self._store_dir)
+        assert not os.path.isdir(teststore.package_path('foo', 'bar'))
+
```

Comment:
pls add corner cases:
- rm on non-existent package
- rm on malformed pkg e.g. with subpkg, empty pkgname

pls ping @eode to add a CLI unittest once this is committed (his CLI unittest framework is in review: https://github.com/quiltdata/quilt-compiler/pull/198

Reply:
Added non-existent package and a sub-package case.